spec/controllers/api/accounts_controller_spec.rb

```diff
@@ -30,6 +39,32 @@
       expect(account1_json['group']).to eq group.id
     end
 
+    it 'returns account with matching name as admin' do
```

Comment:
same here for those tests .... there's only member, non-member role for accounts !